### PR TITLE
Book Model Test

### PR DESCRIPTION
**Pull request by maxy96**
# Summary
* Test book model
   * Validating presence of all attributes
   * Validating format using regex
       *  Genre and author format alphabetic
       *  Year format number and length < 5
# Trello Card
https://trello.com/c/UHIVogG3/19-2-crear-tests-para-el-modelo-de-book

# Screenshot
Captura de Pantalla 2021-06-07 a la(s) 16 54 24

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -1,2 +1,8 @@
 class Book < ApplicationRecord
+  validates_presence_of :genre, :image, :author, :title, :editor, :year
```
Comment (blacksam07) [inline app/models/book.rb]: use a new method with all validations
example
```ruby
validates :genre, :image, :author, :tittle, :editor, :year, presence: :true
```

File: spec/support/database_cleaner.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+RSpec.configure do |config|
```
Comment (blacksam07) [inline spec/support/database_cleaner.rb:1]: you can include this config inside spec_helper

File: spec/support/factory_bot.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
```
Comment (blacksam07) [inline spec/support/factory_bot.rb:1]: inside spec_helper

File: spec/support/shoulda_matchers.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Shoulda::Matchers.configure do |config|
```
Comment (blacksam07) [inline spec/support/shoulda_matchers.rb:1]: inside spec_helper

Comment (maxy96) [reply spec/support/shoulda_matchers.rb:1]: then I should comment the following line `Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }` ?

Comment (blacksam07) [reply spec/support/shoulda_matchers.rb:1]: Nice, my bad i didn’t see the line

File: spec/factories/book.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    genre { 'Suspenso' }
```
Comment (blacksam07) [inline spec/factories/book.rb]: please use https://github.com/faker-ruby/faker

**Review by blacksam07 — CHANGES_REQUESTED** [review]

File: Gemfile [hunk 6]
```diff
@@ -83,7 +92,7 @@ group :development, :test do
 
   gem 'fictium'
 
-  gem 'byebug'
+  gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
```
Comment (blacksam07) [inline Gemfile]: this option is only if you getting an uninitialized constant error

Comment (maxy96) [reply Gemfile]: then no need to add it?

Comment (blacksam07) [reply Gemfile]: use a normal way
```ruby
gem 'faker'
```

Comment (maxy96) [reply Gemfile]: ok!

**Review by blacksam07 — APPROVED** [review]

**Pull request merged**